IDBOOKAPI/IDBOOKAPI/otp_utils.py

```
@@ -1,10 +1,10 @@
 from django.conf import settings
 import math, random
 
+
 def generate_otp(no_digits=4):
     digits = "0123456789"
     OTP = ""
     for i in range(no_digits):
```

Ruff (0.14.8)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)